bookserver/routers/assessment.py

```diff
@@ -85,17 +85,21 @@ async def get_assessment_results(
 # If you just try to specify the two fields as parameters it expects
 # them to be in a query string.
 class HistoryRequest(BaseModel):
+    # `acid` : id of the active code block also called div_id
```

Comment:
Use ``acid`` for code, since `acid` means "look for a reference named acid" instead of the Markdown-style "make this a fixed-width font".